devel/localhost-to-dockerhub

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#
+# Push the nextstrain/base and nextstrain/base-builder images from the local registry to Docker Hub.
+#
+# This pushes just the provided tag. If "latest" is provided as a second argument,
+# the provided tag will also be pushed as "latest"
+#
+set -euo pipefail
+
+if [[ $# -lt 1 ]]; then
+    echo "Please provide a tag." >&2
+    exit 1
+fi
+
+tag=$1
+
+BASE_IMAGE="nextstrain/base"
+BASE_BUILDER_IMAGE="nextstrain/base-builder"
+
+# copy local $tag to remote $tag
+skopeo copy --multi-arch=all --src-tls-verify=false docker://localhost:5000/$BASE_IMAGE:$tag docker://docker.io/$BASE_IMAGE:$tag
+skopeo copy --multi-arch=all --src-tls-verify=false docker://localhost:5000/$BASE_BUILDER_IMAGE:$tag docker://docker.io/$BASE_BUILDER_IMAGE:$tag
```

Comment:
This [errored](https://github.com/nextstrain/docker-base/actions/runs/3055446154/jobs/4928511381), likely transient:

> level=fatal msg="copying image 2/2 from manifest list: trying to reuse blob sha256:828da3c2b6e5dfebc31218a341a7d947edf0fd0c700eb6241afe12ffc1ad11a1 at destination: Head \"https://production.cloudflare.docker.com/registry-v2/docker/registry/v2/blobs/sha256/82/828da3c2b6e5dfebc31218a341a7d947edf0fd0c700eb6241afe12ffc1ad11a1/data?verify=1663189957-ErKuGjE6SA66GsXxUQk9uagRVyw%3D\": net/http: TLS handshake timeout"

Seems like a network error on either [Docker](https://status.docker.com/) or [GitHub](https://www.githubstatus.com/history)'s end. Looking at the linked status pages, I don't see anything documented recently that would have caused this. Just noting here for reference.

Reply:
Retry succeeded.